src/leaflike/utils.clj

```diff
@@ -0,0 +1,21 @@
+(ns leaflike.utils
+  (:import [java.util Date TimeZone]
+           [java.text SimpleDateFormat]
+           [java.sql Timestamp]))
+
+(def email-pattern #"[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?")
+
+(def alpha-num-pattern #"^[a-zA-Z]+$")
+
+(defn get-timestamp
```

Comment:
This returns a Timestamp for the current moment i.e. now, so get-current-timestamp might be a better name